docs/tutorials/Raster-data-WGS84.md

```diff
@@ -42,27 +52,22 @@ var placement = {
 var view = new itowns.GlobeView(viewerDiv, placement);
 ```
 
-Three things are done here. First we get the element of the page on which the
-view will be displayed. However, getting this is not sufficient in our case to
-display a `GlobeView`. [The documentation]{@link GlobeView} specifies that a
-second parameter needs to be present: an object that will help to place the camera
-on the globe.
-
-This object needs to contain two properties : a position which the camera is 
-facing (`coord`), and a camera distance to target coordinates (`range`) in meters.
-For the position argument, we can create a `{@link Coordinates}` object in the WGS84 system -
-whose EPSG code is 4326.
+First we get the element of the page on which the view will be displayed. Then, we create an object that contains camera
+initial placement information. This object should contain two properties : a position which the camera is 
```

Comment:
Maybe we could remove the "should", since rigorously no parameter is mandatory in a `GlobeView` `placement` ?

Reply:
yep, done